pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -166,8 +176,10 @@
     },
     "runtime.any.System.Reflection.Extensions": {
       "StableVersions": [
-        "4.0.1"
+        "4.0.1",
+        "4.3.0"
       ],
+      "BaselineVersion": "4.3.0",
```

Comment:
It's not built in master any more, but is built in 1.1.0.  This was a programmatic merge so it picked up the baseline from 1.1.0.  That's benign since its not building in master.